compiler-rt/test/radsan/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+set(RADSAN_LIT_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
I have a hunch it would be worth adding a comment at the top of this file to explain that the lit tests will be added in a subsequent PR, and that it's just a stub for now